code/__DEFINES/jobs.dm

```diff
@@ -1,29 +1,42 @@
 #define JOB_DISPLAY_ORDER_DEFAULT 0
 
-#define JOB_DISPLAY_ORDER_CAPTAIN 				1
-#define JOB_DISPLAY_ORDER_EXECUTIVE_OFFICER 	2
-#define JOB_DISPLAY_ORDER_STAFF_OFFICER 		3
-#define JOB_DISPLAY_ORDER_PILOT_OFFICER 		4
-#define JOB_DISPLAY_ORDER_TANK_CREWMAN 			5
-#define JOB_DISPLAY_ORDER_CORPORATE_LIAISON 	6
-#define JOB_DISPLAY_ORDER_SYNTHETIC 			7
-#define JOB_DISPLAY_ORDER_AI 					8
-#define JOB_DISPLAY_ORDER_CHIEF_MP 				9
-#define JOB_DISPLAY_ORDER_MILITARY_POLICE 		10
-#define JOB_DISPLAY_ORDER_CHIEF_ENGINEER 		11
-#define JOB_DISPLAY_ORDER_SHIP_TECH     		12
-#define JOB_DISPLAY_ORDER_REQUISITIONS_OFFICER 	13
-#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER 15
-#define JOB_DISPLAY_ORDER_DOCTOR 				16
-#define JOB_DISPLAY_ORDER_MEDIAL_RESEARCHER 	17
-#define JOB_DISPLAY_ORDER_SQUAD_LEADER 			18
-#define JOB_DISPLAY_ORDER_SQUAD_SPECIALIST 		19
-#define JOB_DISPLAY_ORDER_SQUAD_SMARTGUNNER 	20
-#define JOB_DISPLAY_ORDER_SQUAD_CORPSMAN 		21
-#define JOB_DISPLAY_ORDER_SUQAD_ENGINEER 		22
-#define JOB_DISPLAY_ORDER_SQUAD_MARINE 			23
+#define JOB_DISPLAY_ORDER_XENO_QUEEN			1
+#define JOB_DISPLAY_ORDER_XENOMORPH				2
+#define JOB_DISPLAY_ORDER_CAPTAIN				3
+#define JOB_DISPLAY_ORDER_EXECUTIVE_OFFICER		4
+#define JOB_DISPLAY_ORDER_STAFF_OFFICER			5
+#define JOB_DISPLAY_ORDER_PILOT_OFFICER			6
+#define JOB_DISPLAY_ORDER_TANK_CREWMAN			7
+#define JOB_DISPLAY_ORDER_CORPORATE_LIAISON		8
+#define JOB_DISPLAY_ORDER_SYNTHETIC				9
+#define JOB_DISPLAY_ORDER_AI					10
+#define JOB_DISPLAY_ORDER_CHIEF_MP				11
+#define JOB_DISPLAY_ORDER_MILITARY_POLICE		12
+#define JOB_DISPLAY_ORDER_CHIEF_ENGINEER		13
+#define JOB_DISPLAY_ORDER_SHIP_TECH				14
+#define JOB_DISPLAY_ORDER_REQUISITIONS_OFFICER	15
+#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER	16
+#define JOB_DISPLAY_ORDER_DOCTOR				17
+#define JOB_DISPLAY_ORDER_MEDIAL_RESEARCHER		18
+#define JOB_DISPLAY_ORDER_SQUAD_LEADER			19
+#define JOB_DISPLAY_ORDER_SQUAD_SPECIALIST		20
+#define JOB_DISPLAY_ORDER_SQUAD_SMARTGUNNER		21
+#define JOB_DISPLAY_ORDER_SQUAD_CORPSMAN		22
+#define JOB_DISPLAY_ORDER_SUQAD_ENGINEER		23
+#define JOB_DISPLAY_ORDER_SQUAD_MARINE			24
+#define JOB_DISPLAY_ORDER_SURVIVOR				25
 
 #define JOB_FLAG_SPECIALNAME (1<<0)
+#define JOB_FLAG_LATEJOINABLE (1<<1) //Can this job be selected for prefs to join as?
+#define JOB_FLAG_ROUNDSTARTJOINABLE (1<<2) //Joinable at roundstart
+#define JOB_FLAG_NOHEADSET (1<<3) //Doesn't start with a headset on spawn.
+#define JOB_FLAG_ALLOWS_PREFS_GEAR (1<<4) //Allows preference loadouts.
+#define JOB_FLAG_PROVIDES_BANK_ACCOUNT (1<<5) //$$$
```

Comment:
Are banks used?

Reply:
They are sort of functional. Left that only for the Corporate Liaison because /tg/ also has a pretty decent economy system which could be eventually ported for alternate game modes.